### DISCUSSION TITLE
Download in parts

### DISCUSSION THREAD

**Discussion by Travillion** [mkb79/audible-cli#147] [Q&A]
Hello,

I am trying to download a compilation book (includes books 1-4 in a series). Running audible-cli I get the following error:

error: Error downloading location/title.aax. Wrong content type. Expected type(s): ['audio/aax', 'audio/vnd.audible.aax', 'audio/audible']; Got: text/html;charset=UTF-8; Message: This title is not supported for full audio file download, please download individual parts.

What I am inferring is that the title is too big to download at once and needs to be downloaded in parts. I cannot find anything in the -help for how to accomplish this. Is this possible?

**Comment by mkb79** [comment]
Hi. Sorry for my late reply. Downloading audiobook parts is possible by the API but currently not implemented. I'll think about an option to support this.

**Comment by Travillion** [comment]
Thank you for your response. I understand that it's a niche feature. I
can't imagine anyone preferring to download in parts, and this was the
first time I'd ever been forced to. What I ended up doing was downloading
in parts through the Audible website, combining the parts into one file,
getting the chapter json through audible-cli, and manually running the
update chapter python script to get a new chapter metadata file that could
be applied. So in the end, it all worked out okay, but it was definitely a
lot more work.
